package.json

```diff
@@ -49,6 +49,8 @@
     "stringify-object": "^3.2.2"
   },
   "devDependencies": {
+    "babel-core": "^6.26.3",
```

Comment:
peer dep of babel-jest (due to us supporting both babel 6 and 7)

Reply:
Do you think it would be possible for us to remove babel from the project? I don't mind not using `import` but being more strict on what features are allowed. Last time it confused me a lot since tests are relying on it but source code doesn't

Reply:
Yeah, probably. Especially if you start using node 8, as you have object spread and async await out of the box 🙂

Reply:
Yes, exactly. Here is the issue to track #508 ;)